packages/amplify-material-ui/src/auth/sign-up.tsx

```diff
@@ -51,28 +51,48 @@ const signUpFields = [
   },
 ];
 
+export type SignUpValues = Record<string, string>;
+
+export interface SignUpConfig {
+  signUpFields: SignUpField[];
+  initialValues: SignUpValues;
+}
+
+export interface SignUpField {
```

Comment:
We should allow parameters for ux settings here:
- autoComplete
- autoFocus

Reply:
These should be supported now that I'm using `{...inputProps}` on the Field component.

Reply:
Yes that is a good solution